workflow-service/src/test/java/com/redhat/parodos/workflow/WorkFlowDelegateTest.java

```diff
@@ -87,9 +91,19 @@ class WorkFlowDelegateTest {
 					.build()))
 			.build();
 
+	private WorkFlowDefinitionResponseDTO workFlowDefinitionResponseDTO = WorkFlowDefinitionResponseDTO
```

Comment:
I would move this to a, sampleWorkFlowDEfintinoREsponse() function, so if any other test modified this, it will not be flake.